### ISSUE TITLE
resolve a few shiny static import issues

### ISSUE THREAD

**Issue by machow** [machow/quartodoc#66]
- [ ] : events - griffe can't find it, since it's imported in `shiny.__init__` using a star import.
- [ ] : ui.update_text_area() is a the result of assigning `ui.update_text` to a variable, but is read as an Attribute.

```python
from quartodoc import get_object

get_object("shiny", "ui.update_text_area").target
#  <Attribute('update_text_area', 856, 856)>

**Comment by machow** [comment]
For `ui.update_text_area()`, it seems like a function like this might help:

* takes the griffe Function object for the original function (`update_text`) and
* uses its signature and docstring, then
* converts the Attribute for `update_text_area()` to a Function with that sig and docstring, but the line info for the Attribute

**Comment by machow** [comment]
Fixed by #64
